config/initializers/job_configurations.rb

```diff
@@ -182,6 +182,12 @@
         cron: cron_24h,
         args: -> { [Time.zone.today] },
       },
+      # Send reminder letters for old, outstanding GPO verification codes
+      send_gpo_code_reminders: {
+        class: 'GpoReminderJob',
+        cron: cron_24h,
+        args: -> { [Time.zone.now - 14.days] },
```

Comment:
could make this a little shorter with:
```suggestion
        args: -> { [14.days.ago] },
```

Reply:
Changed.